ci-operator/step-registry/openshift/upgrade/vsphere/migration-optin/openshift-upgrade-vsphere-migration-optin-workflow.yaml

```diff
@@ -0,0 +1,23 @@
+workflow:
+  as: openshift-upgrade-vsphere-migration-optin
+  steps:
+    pre:
+    - chain: ipi-vsphere-pre-stableinitial
+    - ref: storage-data-create
```

Comment:
I was thinking about some env. vars that would tell `storage-data-create/check/destroy` what StatefulSets to involve in the step:

```
pre:
    - ref: storage-data-create
      env: STATEFULSETS="1 2"
test:
    - ref: storage-data-check
      env: STATEFULSETS="1 2"

    - ref: storage-conf-migration-optin
    - ref: storage-data-check
      env: STATEFULSETS="1 2"
    - ref: storage-data-destroy
      env: STATEFULSETS="1"
    - ref: storage-data-create
      env: STATEFULSETS="3"

    - ref: <some upgrade step>
    - ref: storage-data-check
      env: STATEFULSETS="2 3"
    - ref: storage-data-destroy
      env: STATEFULSETS="2 3"
...
```